yasser_sharabati/task.py

```diff
@@ -0,0 +1,73 @@
+import json
+
+class Task:
+    task_id_counter = 0
```

Comment:
Assuming we're using a database where the counter resets to 0 with each run, this would result in duplicate IDs, which is incorrect.

Suggestions:

Use a UUID, which is a generated unique string.
Store the last counter value in a file and start from there on the next run.
I recommend using the first option